authenticate-me/package.json

```diff
@@ -20,5 +20,8 @@
   "bugs": {
     "url": "https://github.com/Marielvacacruz/meetup-portfolio-project/issues"
   },
-  "homepage": "https://github.com/Marielvacacruz/meetup-portfolio-project#readme"
+  "homepage": "https://github.com/Marielvacacruz/meetup-portfolio-project#readme",
+  "dependencies": {
+    "pg": "^8.7.3"
```

Comment:
This should be a package in the backend/package.json rather than in the root package.json